### PR TITLE
WIP: BZ1935005 - Copy approved multipath changes from downstream to upstream

### PR DESCRIPTION
**Pull request by dcdacosta**
Fixes issues:

1. https://bugzilla.redhat.com/show_bug.cgi?id=1935005
2. https://bugzilla.redhat.com/show_bug.cgi?id=2018353

Changes proposed in this pull request:

- Corrected the path for multipath drop-in config files
- These changes were completed and approved in the RHV repo on GitLab and I'm copying them over now that we are working upstream first.

I confirm that this pull request was submitted according to the contribution guidelines: (@dcdacosta)

This pull request needs review by: Already reviewed and approved by SME, QE and peer reviewers.

### CODE REVIEW

File: source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc [hunk 1]
```diff
@@ -30,9 +30,9 @@ Not following these guidelines can cause catastrophic storage errors.
 
 .Procedure
 
-. Create a new configuration file in the `/etc/multipath/conf.d/user.conf` directory.
+. Create a new configuration file in the `/etc/multipath/conf.d` directory.
 
-. Copy the individual setting you want to override from `/etc/multipath.conf` to the new configuration file in `/etc/multipath/conf.d/user.conf`. Remove any comment marks, edit the setting values, and save your changes.
+. Copy the individual setting you want to override from `/etc/multipath.conf` to the new configuration file in `/etc/multipath/conf.d/<mydevice>.conf`. Remove any comment marks, edit the setting values, and save your changes.
```
Comment (stoobie) [inline source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc:35]:
```suggestion
. Copy the individual setting you want to override from `/etc/multipath.conf` to the new configuration file in `/etc/multipath/conf.d/<my_device>.conf`. Remove any comment marks, edit the setting values, and save your changes.
```
See https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

File: source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc [hunk 2]
```diff
@@ -47,11 +47,12 @@ Do not restart the multipathd service. Doing so generates errors in the VDSM log
 
 .Verification steps
 
-Test the new configuration performs as expected on a non-production cluster in a variety of failure scenarios. For example, disable all of the storage connections. Then enable one connection at a time and verify that doing so makes the storage domain reachable.
+. Test the new configuration performs as expected on a non-production cluster in a variety of failure scenarios. For example, disable all of the storage connections.
```
Comment (stoobie) [inline source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc:50]:
```suggestion
. Test that the new configuration performs as expected on a non-production cluster in a variety of failure scenarios. For example, disable all of the storage connections.
```

File: source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc [hunk 3]
```diff
@@ -47,11 +47,12 @@ Do not restart the multipathd service. Doing so generates errors in the VDSM log
...
+. Enable one connection at a time and verify that doing so makes the storage domain reachable.
 
 .Additional resources
 
-*  link:{URL_virt_product_docs}{URL_format}installing_red_hat_virtualization_as_a_self-hosted_engine_using_the_cockpit_web_interface#ref-Recommended_Settings_for_Multipath_conf_SHE_cockpit_deploy[Recommended Settings for Multipath.conf]
+* xref:ref-Recommended_Settings_for_Multipath_conf_{context}[Recommended Settings for Multipath.conf]
```
Comment (stoobie) [inline source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc:55]: replace {context} with whatever {context} evaluates to.

For example, if the target of the xref is in a location in the document where the value of {context} is `snoopy`, then this xref should be:
` xref:ref-Recommended_Settings_for_Multipath_conf_snoopy`

**Review by stoobie — CHANGES_REQUESTED** [review]
A few minor changes needed.

File: source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc (hunk 3 above)
Comment (dcdacosta) [reply source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc:55]: @stoobie See email for questions about this cross reference. Thanks!

File: source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc (hunk 1 above)
Comment (dcdacosta) [reply source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc:35]: Fixed.

File: source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc (hunk 2 above)
Comment (dcdacosta) [reply source/documentation/common/storage/proc-Customizing_Multipath_Configurations_for_SAN_Vendors.adoc:50]: Fixed.

**Pull request closed without merging**